src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is
+      {` ${person.name}`}
```

Comment:
```suggestion
      My name is {name}
```

Reply:
It'll be lint mistake. {name} requires new line. And it misses spaces then